optimize.Dockerfile

```diff
@@ -73,7 +73,9 @@ VOLUME /tmp
 
 # Switch to root to allow setting up our own user
 USER root
-RUN mkdir -p /usr/local/bin/ && \
+RUN apk update && \
+    apk add --no-cache bash netcat-openbsd wget && \
```

Comment:
This change is not going to work since the new hardened base images from Minimus do not allow installing OS packages after the build - we instead do add packages via the https://docs.minimus.io/advanced-tooling/image-creator feature. We can discuss the [details on Slack](https://camunda.slack.com/archives/C08NZ7E9ZT2/p1764243598694899)